guides/common/modules/snip_prerequisite-project-client-repository.adoc

```diff
@@ -0,0 +1,2 @@
+* {project-client-name} repository for the operating system version of the host is synchronized on {ProjectServer} and enabled in the activation key you use.
```

Comment:
@ekohl Help me out here, please, if you can. Will this wording work also for hosts that are already registered? Assuming the host has an AK assigned, can I tell the user to enable the repo in the AK and it will available on the host without further steps?

I might be overthinking this, but I got stuck on it :laughing:

Reply:
I'm not an expert on content and normally look at @jeremylenz or @ianballou.

Normally I'd say something like this is at least needed.
```suggestion
ifdef::foreman-el[]
* {project-client-name} repository is available on the host.
endif::[]
ifndef::foreman-el[]
* {project-client-name} repository for the operating system version of the host is synchronized on {ProjectServer} and enabled in the activation key you use.
endif::[]
```

And the link below is also invalid for `foreman-el`.

Reply:
@Lennonka No, if a host is already registered, changes to its activation key will have no effect on the host. I'd suggest maybe just "enabled for the host". I'm on the fence as to whether more elaboration is needed here about content overrides.

Reply:
@jeremylenz Thank you! The content overrides would be [this one](https://docs.theforeman.org/nightly/Managing_Content/index-katello.html#Changing_the_Repository_Sets_Status_for-a-Host-in_foreman_content-management), right?

Reply:
Yep, that's where you'd set content overrides for a host. (They can also be set on activation keys, but note my original comment above.)